components/badge/badge.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="badge.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
```

Comment:
You can import fontawesome in your CSS so you don't have to import it multiple times on every HTML page. 

Also, you can use a single CSS file for all the HTML files, import all your component CSS into one file, and attach it to all the HTML that way you will get the error if by mistake overwrite somewhere.